lib/deas/test_helpers.rb

```diff
@@ -1,12 +1,16 @@
+require 'rack/request'
+require 'rack/response'
 require 'deas/test_runner'
 
 module Deas
 
   module TestHelpers
 
-    module_function
```

Comment:
@jcredding yeah, we don't use it and I wanted it to be consistent with our other interface gems.  Like we discussed, I say let's leave it out for now and bring it back if/when we have a need for it.